ArcGISRuntimeSDKQt_QMLSamples/Geometry/FormatCoordinates/FormatCoordinates.qml

```diff
@@ -0,0 +1,247 @@
+                    style: Enums.SimpleMarkerSymbolStyleX
+                    color: Qt.rgba(1.0, 0.0, 0.0, 1.0)
+                    size: 15.0
+                }
+            }
+
+            Graphic {
+                id: locationGraphic
+                geometry: Point {
+                    x: -117.195723
+                    y: 34.056195
+                    spatialReference: SpatialReference {
+                        wkid: 4326
+                    }
+                }
+                onComponentCompleted: {
+                    setTextFromPoint(locationGraphic.geometry);
+                }
+            }
+        }
+
+        onMouseClicked: {  // on MapView
+            handleLocationUpdate(mouse.mapPoint);
+        }
+    }
+
+    Rectangle {
+        id: leftColumnRect
+
+        anchors {
+            left: parent.left
+            bottom: parent.bottom
+            right: rightColRect.left
+        }
+        height: 140 * scaleFactor
+        width: labelWidth
+
+        Text {
+            id: labelDD
+            anchors {
+                top: parent.top
+                right: parent.right
+            }
+            font.pixelSize: fontPixelSize
+            padding: textPadding
+            horizontalAlignment: Text.AlignRight
+            text: qsTr("Decimal Degrees:  ")
+        }
+
+        Text {
+            id: labelDMS
+            anchors {
+                top: labelDD.bottom
+                right: parent.right
+            }
+            font.pixelSize: fontPixelSize
+            padding: textPadding
+            horizontalAlignment: Text.AlignRight
+            text: qsTr("Degrees Minutes Seconds:  ")
+        }
+
+        Text {
+            id: labelUtm
+            anchors {
+                top: labelDMS.bottom
+                right: parent.right
+            }
+            font.pixelSize: fontPixelSize
+            padding: textPadding
+            horizontalAlignment: Text.AlignRight
+            text: qsTr("UTM:  ")
+        }
+
+        Text {
+            id: labelUsng
+            anchors {
+                top: labelUtm.bottom
+                right: parent.right
+            }
+            font.pixelSize: fontPixelSize
+            padding: textPadding
+            horizontalAlignment: Text.AlignRight
+            text: qsTr("USNG:  ")
+        }
+    }
+
+    Rectangle {
+        id: rightColRect
+
+        anchors {
+            top: leftColumnRect.top
+            left: leftColumnRect.right
+            bottom: parent.bottom
+            right: parent.right
+        }
+        height: leftColumnRect.height
+        width: coordinateTextWidth + (10 * scaleFactor)
+
+        TextField {
+            id: textDD
+            anchors {
+                top: parent.top
+                left: parent.left
+            }
+            width: coordinateTextWidth
+            font.pixelSize: fontPixelSize
+            onAccepted: {
+                handleTextUpdate("Decimal Degrees", text);
```

Comment:
you can't really define an enum in QML, but maybe you could have a string you use throughout for each of these strings, as they are all used a few times